app/views/publish/courses/_basic_details_tab.html.erb

```diff
@@ -146,14 +146,14 @@
             # visually_hidden_text: "date course starts",
           }) %>
         <% else %>
-          <% row.action() %>
+          <% row.action %>
         <% end %>
       <% end %>
 
       <% summary_list.row(html_attributes: { data: { qa: "course__name" } }) do |row| %>
         <% row.key { "Title" } %>
         <% row.value { course.name } %>
-        <% row.action() %>
+        <% row.action %>
```

Comment:
is this a safe change?

Reply:
I think so, ruby shouldn't need the brackets. For that `govuk_summary_list` thing, the [proper way to skip actions](https://design-system.service.gov.uk/components/summary-list/) is to change the class for the row to include `govuk-summary-list__row--no-actions` to the element class. But because we have dynamic decisions on whether there's an action, it looks like it's just easier to say there's an action, but provide no content. 

To be honest, I'm not sure what the better way of doing it is, but removing the brackets doesn't seem to break it :D